src/lightning/fabric/utilities/testing/_runif.py

```diff
@@ -27,7 +27,7 @@
 from lightning.fabric.utilities.imports import _TORCH_GREATER_EQUAL_2_1
 
 
-def _RunIf(
+def _run_if_reasons(
```

Comment:
"Run if reasons" doesn't mean the same as "RunIf reasons"

```suggestion
def _runif_reasons(
```